drivers/mlx90393/include/mlx90393_params.h

```diff
@@ -0,0 +1,205 @@
+/*
+ * Copyright (C) 2023 mi6527ri
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     drivers_mlx90393
+ *
+ * @{
+ * @file
+ * @brief       Default configuration for MLX90393 device driver
+ *
+ * @author      Michael Ristau <michael.ristau@fh-erfurt.de>
+ */
+
+#ifndef MLX90393_PARAMS_H
+#define MLX90393_PARAMS_H
+
+#include "board.h"
+#include "mlx90393.h"
+#include "mlx90393_constants.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @name    Set default configuration parameters for the MLX90393
+ * @{
+ */
+#ifndef MLX90393_PARAM_MODE
+/**
+ * @brief   Default measurement mode
+ */
+#define MLX90393_PARAM_MODE             (MLX90393_MODE_BURST)
+#endif
+
+#if IS_USED(MODULE_MLX90393_INT) || DOXYGEN
+#ifndef MLX90393_PARAM_INT_PIN
+/**
+ * @brief   Default interrupt pin
+ */
+#define MLX90393_PARAM_INT_PIN          .int_pin = (GPIO_PIN(PORT_C, 8)),
+#endif
+#else
+#define MLX90393_PARAM_INT_PIN
+#endif
+
+#ifndef MLX90393_PARAM_GAIN
+/**
+ * @brief   Default gain
+ */
+#define MLX90393_PARAM_GAIN             (MLX90393_GAIN_1X)
+#endif
+
+#ifndef MLX90393_PARAM_RES
+/**
+ * @brief   Default resolution
+ */
+#define MLX90393_PARAM_RES              (MLX90393_RES_19)
+#endif
+
+#ifndef MLX90393_PARAM_ODR
+/**
+ * @brief   Default sampling rate
+ */
+#define MLX90393_PARAM_ODR              (MLX90393_ODR_10HZ)
+#endif
+
+#ifndef MLX90393_PARAM_OSR
+/**
+ * @brief   Default oversampling ratio
+ */
+#define MLX90393_PARAM_OSR                          \
+{                                                   \
+    .mag            = MLX90393_OSR_1,               \
+    .temp           = MLX90393_OSR_1                \
+}
+#endif
```

Comment:
Das Gleiche wie bei der Definition des Interrupt-Pin, sowas kann man nicht überschreiben, zumindest nicht an der Make-Kommandozeile. Daher wie folgt:

```suggestion
#ifndef MLX90393_PARAM_OSR_MAG
#define MLX90393_PARAM_OSR_MAG          MLX90393_OSR_1
#endif

#ifndef MLX90393_PARAM_OSR_TEMP
#define MLX90393_PARAM_OSR_TEMP         MLX90393_OSR_1
#endif

/**
 * @brief   Default oversampling ratio
 */
#define MLX90393_PARAM_OSR                          \
{                                                   \
    .mag            = MLX90393_PARAM_OSR_MAG,       \
    .temp           = MLX90393_PARAM_OSR_TEMP,      \
}
```